tests/unittests/benchmark/test_benchmark.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for :mod:`orion.benchmark.Benchmark`."""
+
+import plotly
+import pytest
+
+import orion.core.io.experiment_builder as experiment_builder
+from orion.benchmark import Benchmark, Study
+from orion.benchmark.assessment import AverageRank, AverageResult
+from orion.benchmark.task import CarromTable, RosenBrock
+from orion.client.experiment import ExperimentClient
+from orion.core.worker.experiment import Experiment
+from orion.testing import OrionState
+
+
+def build_study_experiments(algo_num, task_num):
```

Comment:
This should also be in orion.testing.

Reply:
do we have a criterion for `orion.testing`, compared with what did with `conftest` directly in the test code?

Reply:
We use conftest for fixtures and orion.testing for functions that are widely useful. orion.testing is also useful to reuse testing code in external plugins. Since the creation of the OrionState class to ease testing when using the database, we try to favor functions to populate the DB rather than fixtures. This way we can also parametrize the function that creates the DB content. It's more tedious to do with fixtures. For the rest, fixtures work well.